manifests/prometheus/alerts.d/paas-billing-costs.yml

```
@@ -46,9 +49,10 @@
     name: PaaSBillingAppCostsFallsShortOfEC2Costs
     rules:
       - alert: PaaSBillingAppCostsFallsShortOfEC2Costs
-        expr: sum(increase(paas_billing_total_costs_pounds{name="app"}[24h])) / sum(paas_aws_cost_explorer_by_service_pounds{service=~"^EC2.+"}) < .9
+        expr: sum(increase(paas_billing_total_costs_pounds{name="app"}[24h])) / sum(paas_aws_cost_explorer_by_service_pounds{service=~"Amazon Elastic Compute Cloud - Compute"}) < .9
```

```suggestion
        expr: sum(increase(paas_billing_total_costs_pounds{name="app"}[24h])) / sum(paas_aws_cost_explorer_by_service_pounds{service=~"(Amazon Elastic Compute Cloud - Compute)|(^EC2.+)"}) < .9
```

I would err on the safe side with ⬆️ but theoretically we shouldn't care about "EC2 - Other"

:shipit:

---

I think "EC2 - Other" is small enough to ignore, and it keeps the thing simpler just having one.

If the metric changes its name then we'll notice fairly quickly.